### PR TITLE
Recommended section added

### PR DESCRIPTION
**Pull request by jalamirzayeva**

### CODE REVIEW

File: src/layouts/HomePage/HomePageMain/MainBestSellers/MainBestSellers.scss [hunk 1]
```diff
@@ -11,7 +15,7 @@
             border-radius: 20px;
             border: 1px solid #50525436;
             .img{
-                background-color: #50525436;
+                background: #50525436;
```
Comment (Negodyaj) [inline src/layouts/HomePage/HomePageMain/MainBestSellers/MainBestSellers.scss:18]: you must use variables for colors)

File: src/layouts/HomePage/HomePageMain/MainRecommended/MainRecommended.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        display: flex;
+        padding: 40px 0px;
+        .carousel-card{
+            display: flex;
+            flex-direction: column;
+            width: 242px;
+            border-radius: 20px;
+            border: 1px solid #50525436;
+            .img{
+                background: #50525436;
```
Comment (Negodyaj) [inline src/layouts/HomePage/HomePageMain/MainRecommended/MainRecommended.scss:18]: variables

File: src/layouts/HomePage/HomePageMain/MainRecommended/MainRecommended.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+           > 
+            <div className="carousel-card">
+              <div className="img">
+                <img src={carouselimg1} alt="" />
+              </div>
+              <div className="desc">
+                <h5>iPhone 13 mini 128 GB | Pink (çəhrayı)</h5>
+                <p className="price-product">null</p>
+              </div>
+            </div>
```
Comment (Negodyaj) [inline src/layouts/HomePage/HomePageMain/MainRecommended/MainRecommended.tsx:48]: you must create the CarouselCard component for the "carousel-card" div.
seems like the same cards are used in MainBestSellers component, so CarouselCard must be a shared component

Comment (jalamirzayeva) [reply src/layouts/HomePage/HomePageMain/MainRecommended/MainRecommended.tsx:48]: okay i fixed all

**Pull request merged**